src/app/loadout/known-values.ts

```diff
@@ -22,12 +22,10 @@ export const slotSpecificPlugCategoryHashes = [
   armor2PlugCategoryHashesByName.classitem,
 ];
 
-// TODO generate this somehow so we dont need to maintain it
-export const raidPlugCategoryHashes = [
-  PlugCategoryHashes.EnhancementsSeasonOutlaw, // last wish
-  PlugCategoryHashes.EnhancementsRaidGarden, // garden of salvation
-  PlugCategoryHashes.EnhancementsRaidDescent, // deep stone crypt
-  PlugCategoryHashes.EnhancementsRaidV520, // vault of glass
-];
+export const raidPlugCategoryHashes: number[] =
+  Array.isArray(raidModPlugCategoryHashes) &&
+  raidModPlugCategoryHashes.every((pch) => typeof pch === 'number')
```

Comment:
When is it not an array of numbers?

Reply:
![image](https://user-images.githubusercontent.com/68782081/123064194-e8ee9c80-d3c2-11eb-8282-bfcaa4e49fe1.png)
![image](https://user-images.githubusercontent.com/68782081/123064371-13405a00-d3c3-11eb-9837-eaf67f6c4ea9.png)

yeah don't think this verification is needed, as TS is already aware of this structure's types

Reply:
Yeah I am probably being overly cautious in the event something changes in an update. Feels weird importing json as an object. I will update it accordingly.

Reply:
Done